### DISCUSSION TITLE
Want to use already created PrivateDNSZone in Private AKS

### DISCUSSION THREAD

**Discussion by bansioza44** [Azure/ResourceModules#3121] [Q&A] [answered]
Hi Team, 
in container service's managed cluster module recently been change from allowing usePrivateDNSZone as string to bool.
For our requirement we want to use existing private DNS Zone Resource ID. How we leverage this now?

**Comment by eriqua** [comment, marked as answer]
Hi @bansioza44, thanks for opening this discussion and sorry for the late reply. Discussions are currently not monitored promptly and are planned to be disabled in favor of the new issue template [Questions/Feedback].

I'm converting this discussion to an issue in order for the team to be able to triage it.

Please refer to issue #3143  going forward.
